SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialVirtualWorkerTranslator.h

```diff
@@ -72,6 +72,7 @@ class SPATIALGDK_API SpatialVirtualWorkerTranslator
 	// Serialization and deserialization of the mapping.
 	void ApplyMappingFromSchema(Schema_Object* Object);
 	void WriteMappingToSchema(Schema_Object* Object);
+	bool IsValidMapping(Schema_Object* Object);
```

Comment:
I think this and WriteMappingToSchema can be const

Reply:
Done